web-components/src/components/inputs/new/RadioGroup/RadioGroup.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+
+import { Radio } from '../Radio/Radio';
+import { RADIO_PREFERABLE } from '../Radio/Radio.constants';
+import { VideoInput } from '../VideoInput/VideoInput';
+import { RadioGroup } from './RadioGroup';
+
+export default {
+  title: 'atoms/inputs/RadioGroup',
+  component: RadioGroup,
+} as ComponentMeta<typeof RadioGroup>;
+
+const Template: ComponentStory<typeof RadioGroup> = args => {
+  const selectedValue = 'video';
+
+  return (
+    <RadioGroup sx={{ maxWidth: 500 }} {...args}>
+      <>
+        <Radio
+          name={'radio-story'}
+          label={'Add a video'}
+          labelOptionalText={RADIO_PREFERABLE}
```

Comment:
Maybe best to keep the same prop name as the existing one on `FieldFormControl` (that uses `ControlledFormLabel`)? It uses `optional` as prop name, we could use one or the other, I don't have a preference but let's stay consistent.

Reply:
Fair point, let's stay with `optional` then.

Reply:
with the `optional` props you can also put custom text

Reply:
see https://github.com/regen-network/regen-web/blob/9252815680e713ba6fd099d5a1e03ddc07e90e6f/web-components/src/components/form/ControlledFormLabel.tsx#L21